qiime/beta_diversity.py

```diff
@@ -34,7 +34,7 @@
 import warnings
 warnings.filterwarnings('ignore', 'Not using MPI as mpi4py not found')
 
-from numpy import asarray
+from numpy import vstack
```

Comment:
True, no strong feelings either ... Just ignore that comment.

Yoshiki Vázquez-Baeza

> On Jun 4, 2015, at 9:45 PM, Daniel McDonald <notifications@github.com> wrote:
> 
> In qiime/beta_diversity.py:
> 
> > @@ -34,7 +34,7 @@
> >  import warnings
> >  warnings.filterwarnings('ignore', 'Not using MPI as mpi4py not found')
> >  
> > -from numpy import asarray
> > +from numpy import vstack
> no strong feelings here, can do if you'd like but the iteration with jenkins is large
> 
> —
> Reply to this email directly or view it on GitHub.
>